packages/telegram-bot-api/build.sh

```diff
@@ -1,44 +1,17 @@
-TERMUX_PKG_HOMEPAGE=https://github.com/tdlib/telegram-bot-api
+TERMUX_PKG_HOMEPAGE=https://core.telegram.org/bots
 TERMUX_PKG_DESCRIPTION="Telegram Bot API server"
 TERMUX_PKG_LICENSE="BSL-1.0"
-TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_SRCURL=git+https://github.com/tdlib/telegram-bot-api
-_COMMIT=12bbe26692b8c6e347aaf3e32e0e9d11b86bb56d
-_COMMIT_DATE=2024.07.30
-TERMUX_PKG_VERSION=${_COMMIT_DATE//./}
-TERMUX_PKG_SHA256=6c0dddef10c0e1050a0f9840895d71830a867bb1932cd150d1d56b1e9510fc18
-TERMUX_PKG_AUTO_UPDATE=false
-TERMUX_PKG_GIT_BRANCH=master
-TERMUX_PKG_HOSTBUILD=true
+TERMUX_PKG_MAINTAINER="Maria Lisina @Sarisan"
+TERMUX_PKG_VERSION="7.8"
```

Comment:
Epoch should be bumped to 1 as `7.8` < `20240730`

```suggestion
TERMUX_PKG_VERSION="1:7.8"
```